core/src/robot/Robot.cc

```diff
@@ -1184,6 +1253,30 @@ namespace jiminy
         }
     }
 
+    void Robot::computeConstraints(vectorN_t const & q,
+                                   vectorN_t const & v,
+                                   matrixN_t & jacobianOut,
+                                   vectorN_t & driftOut)
+    {
+        // Give the right number of columns to the jacobian.
+        jacobianOut.resize(0, pncModel_.nv);
+
+        // Call computeAllTerms to update frame jacobian: this is needed by forwardDynamics anyway.
+        pinocchio::computeAllTerms(pncModel_, pncData_, q, v);
```

Comment:
I don't like computeAllTerms. It is too high-level for me and I don't understand what it is actually doing. I would rather prefer to call manually each submethod if possible. Unless it is actually necessary to compute "all" terms.